bibliotheca-hertziana-max-planck-institute-for-art-history.csl

```diff
@@ -0,0 +1,317 @@
+    <choose>
+      <if variable="issued">
+        <date variable="original-date" prefix=" (" suffix=") ">
+          <date-part name="year"/>
+        </date>
+        <date variable="issued">
+          <date-part name="year"/>
+        </date>
+      </if>
+      <else>
+        <text term="no date" form="short"/>
+      </else>
+    </choose>
+  </macro>
+  <citation disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true">
+    <sort>
+      <key variable="issued" sort="ascending"/>
+      <key macro="author"/>
+    </sort>
+    <layout delimiter=", ">
+      <group delimiter=":">
+        <group delimiter=" ">
+          <text macro="contributors-short"/>
+          <group delimiter="; ">
+            <text variable="dimensions" prefix="(" suffix=")"/>
+          </group>
+          <text macro="date-reference"/>
+        </group>
+        <text macro="point-locators"/>
+      </group>
+    </layout>
+  </citation>
+  <bibliography et-al-min="12" et-al-use-first="6">
+    <sort>
+      <key variable="author" sort="ascending"/>
+      <key macro="author"/>
+    </sort>
+    <layout>
+      <group delimiter=":" display="block">
+        <group delimiter=" ">
+          <group>
+            <text macro="contributors-short"/>
+          </group>
+          <group delimiter="; ">
+            <text variable="dimensions" prefix="(" suffix=")"/>
+          </group>
+          <text macro="date-reference" suffix=" "/>
+        </group>
+        <text macro="point-locators"/>
+      </group>
+      <group delimiter=" ">
+        <text macro="author"/>
+        <text macro="title"/>
+        <choose>
+          <if type="book" match="any">
+            <group>
+              <text macro="editor"/>
+              <text variable="number-of-volumes" form="short" suffix=" vols.,"/>
+            </group>
+            <group delimiter=" ">
+              <text variable="collection-title" suffix=","/>
+              <text variable="collection-number" suffix=","/>
+              <text macro="edition" prefix=" " suffix=", "/>
+              <text macro="publisher" prefix=" " suffix=","/>
+              <text variable="publisher-place" suffix=","/>
+            </group>
+            <group delimiter=" ">
+              <label strip-periods="true" suffix="." variable="volume" form="short"/>
+              <text variable="volume" suffix=","/>
+            </group>
+            <date date-parts="year" form="text" variable="issued" suffix="."/>
+            <group delimiter=" ">
+              <label strip-periods="true" suffix=". " variable="page" form="short"/>
+              <text variable="page"/>
+            </group>
+            <group>
+              <text variable="URL" prefix="URL: " suffix="."/>
+            </group>
+          </if>
+          <else-if type="article-journal" match="any">
+            <group delimiter=" ">
+              <group delimiter=", ">
+                <group delimiter=": ">
```

Comment:
remove groups with just one element -- they have no effect and make the style harder to read

Reply:
removed